rusty_pizza_server/src/order_model/meals.rs

```diff
@@ -106,4 +128,53 @@ mod tests {
             }
         );
     }
+
+    #[rstest(prices, expected_total,
+        case(vec![Money::new(2, 25), Money::new(5, 50), Money::new(7, 33)], Money::new(15, 08)),
+        case(vec![Money::new(3, 50), Money::new(4, 42)], Money::new(7, 92)),
+    )]
+    fn total_price_is_calculated_correctly(prices: Vec<Money>, expected_total: Money) {
+        //Given
+        let user = Rc::new(User::new(String::from("Peter")));
+        let mut meals = Meals::new(user.clone());
```

Comment:
Cloning the `User` shouldn't be necessary here:
```suggestion
        let mut meals = Meals::new(user);
```

Reply:
You are right. Cloning is not necessary, because the `user` is not reused